pages/transfer.vue

```diff
@@ -85,7 +85,7 @@
           }}{{ '...' }}
         </b-button>
         <b-button
```

Comment:
better would be to have handler that returns 0 for the bsx imo

Reply:
not sure what are you commenting on here